src/main/java/one/util/streamex/EntryStream.java

```diff
@@ -2284,4 +2285,88 @@ public void forKeyValue(BiConsumer<? super K, ? super V> action) {
                 Stream.generate(() -> new SimpleImmutableEntry<>(keySupplier.get(), valueSupplier.get())),
                 StreamContext.SEQUENTIAL);
     }
+
+    /**
+     * Returns a {@code EntryStream} consisting of the elements of this stream those keys are not equal to given keys.

```

Comment:
_an_ EntryStream. Also, `those` -> `whose`. Also, implementation note about performance should be added, like it's done in `StreamEx::without`.